packages/cdk8s-cli/src/cli/cmds/synth.ts

```diff
@@ -25,9 +32,14 @@ class Command implements yargs.CommandModule {
       env: {
         ...process.env,
         CDK8S_OUTDIR: outdir,
+        CDK8S_STDOUT: stdout,
       },
     });
 
+    if (stdout) {
+      process.exit(0);
+    }
```

Comment:
Why is this needed?

Reply:
Skipping checks below this that verify an output file was created